development/dev.env

```diff
@@ -30,5 +30,5 @@ NAUTOBOT_SELENIUM_URL=http://selenium:4444/wd/hub  # WebDriver (Selenium client)
 NAUTOBOT_SELENIUM_HOST=nautobot  # LiveServer (Nautobot server)
 
 # Needed for Celery, should match Nautobot/Redis values above
-CELERY_BROKER_URL="redis://:decinablesprewad@redis:6379/0"
-CELERY_RESULT_BACKEND="redis://:decinablesprewad@redis:6379/0"
+NAUTOBOT_CELERY_BROKER_URL="redis://:decinablesprewad@redis:6379/0"
```

Comment:
since we use `parse_redis_connection(redis_database=X)` for other REDIS URLs does it make sense to do the same here?  Or should we change the others to use something like `NAUTOBOT_CACHEOPS_REDIS` and specify the URL the same way?

Reply:
I was thinking that since we're deferring to the `CACHES` config which was already defined just above, that we could explicitly use that unless otherwise defined?

Reply:
There is no env `CACHES` in nautobot_config `CACHES` also uses the parse_redis_connection which reads the env vars listed above in this file.

Reply:
I see what you mean now. Thanks.